<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFImFCCSRelativeIntensityDialog</class>
 <widget class="QWidget" name="QFImFCCSRelativeIntensityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate Relative CCF Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CCF file:</string>
       </property>
       <property name="buddy">
        <cstring>cmbCCF</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnNextACF">
       <property name="text">
        <string>&amp;next</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/prop_next.png</normaloff>:/lib/prop_next.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QFRDRComboBox" name="cmbCCF"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,1" columnstretch="1,1">
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>rel. CCF amplitude</string>
       </property>
       <property name="buddy">
        <cstring>pltData</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0,0">
       <item row="3" column="2">
        <widget class="QPushButton" name="btnStoreDataAll">
         <property name="text">
          <string>&amp;store all results</string>
         </property>
         <property name="icon">
          <iconset resource="../../lib/lib.qrc">
           <normaloff>:/lib/save_all.png</normaloff>:/lib/save_all.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnStoreData">
         <property name="text">
          <string>&amp;store this result</string>
         </property>
         <property name="icon">
          <iconset resource="../../lib/lib.qrc">
           <normaloff>:/lib/save16.png</normaloff>:/lib/save16.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QFPlotter" name="pltData" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnReplot">
         <property name="text">
          <string>&amp;replot</string>
         </property>
         <property name="icon">
          <iconset resource="../../lib/lib.qrc">
           <normaloff>:/lib/refresh16.png</normaloff>:/lib/refresh16.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QFRDRImageToRunPreview" name="widOverviewCCF" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>overview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/help.png</normaloff>:/lib/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/exit.png</normaloff>:/lib/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFRDRComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFRDRImageToRunPreview</class>
   <extends>QWidget</extends>
   <header>qfrdrimagetorunpreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnReplot</sender>
   <signal>clicked()</signal>
   <receiver>QFImFCCSRelativeIntensityDialog</receiver>
   <slot>replotImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStoreData</sender>
   <signal>clicked()</signal>
   <receiver>QFImFCCSRelativeIntensityDialog</receiver>
   <slot>addResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFImFCCSRelativeIntensityDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>replotImages()</slot>
  <slot>addResult()</slot>
 </slots>
</ui>
